### ISSUE TITLE
Create interface to upload file

### ISSUE THREAD

**Issue by paulozip** [paulozip/alchemy#4]
Instead of use a cURL post to upload new datasets, it would be nice to have and interface where this could be done by the user.

**Comment by millengustavo** [comment]
This issue is resolved here:
https://github.com/paulozip/arauto/pull/16

**Comment by swordfish81** [comment]
Hi, any idea on when the upload feature will be included? By the way, having problems uploading a csv using curl. Would you be able to help? @paulozip

**Comment by charbelmerhej** [comment]
Yes also having issues uploading using curl. I am getting an Error 405: Method Not Allowed.

**Comment by fredzannarbor** [comment]
This should be easy in streamlit. There is a file uploader widget.
